verifier/build.gradle.kts

```diff
@@ -15,10 +15,12 @@ repositories {
 
 dependencies {
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4")
+    implementation("io.github.pdvrieze.xmlutil:serialization:0.84.3")
```

Comment:
What about `serialization-jvm` version? It should contains some addition API, which can be usefull (or may not). WDYT?